email_app/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # MaidSafe Email Tutorial App Change Log
 
+## [0.4.3]
```

Comment:
@hunterlester , let's please start using the new template for the changelog, as an example you can use this: maidsafe/beaker-plugin-safe-app@f15afd8#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR3
Eventually we may not only automate this but also have scripts to verify its correctness.